pkg/controller/job-controller.go

```diff
@@ -230,19 +241,16 @@ func (c *JobController) SyncHandler(key string) (Result, error) {
 	}
 	err = c.k8sClient.Get(ctx, client.ObjectKeyFromObject(tenant), tenant)
 	if err != nil {
-		jobCR.Status.Phase = miniojob.MinioJobPhaseError
-		jobCR.Status.Message = fmt.Sprintf("Get tenant %s/%s error:%v", jobCR.Spec.TenantRef.Namespace, jobCR.Spec.TenantRef.Name, err)
-		err = c.updateJobStatus(ctx, &jobCR)
-		return WrapResult(Result{}, err)
+		return WrapResult(Result{}, fmt.Errorf("get tenant %s/%s error:%w", jobCR.Spec.TenantRef.Namespace, jobCR.Spec.TenantRef.Name, err))
 	}
 	if tenant.Status.HealthStatus != miniov2.HealthStatusGreen {
-		return WrapResult(Result{RequeueAfter: time.Second * 5}, nil)
+		return WrapResult(Result{RequeueAfter: time.Second * 5}, fmt.Errorf("get tenant %s/%s error:%w", jobCR.Spec.TenantRef.Namespace, jobCR.Spec.TenantRef.Name, err))
 	}
 	// check sa
 	pbs := &stsv1beta1.PolicyBindingList{}
 	err = c.k8sClient.List(ctx, pbs, client.InNamespace(namespace))
 	if err != nil {
-		return WrapResult(Result{}, err)
+		return WrapResult(Result{}, fmt.Errorf("get policybinding error:%w", err))
```

Comment:
Same here (also updated description, because you list instead of get):
```suggestion
		return WrapResult(Result{}, fmt.Errorf("list policybinding error: %w", err))
```